### PR TITLE
fixed: Final vehicle summary Report

### PR DESCRIPTION
**Pull request by saurkarankita**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/reports/fleet/vehicle-report/summary/summary.component.html [hunk 1]
```diff
@@ -22,6 +22,8 @@
                             placeholder="Select status">
                             <ng-option value="active">Active</ng-option>
                             <ng-option value="inActive">Inactive</ng-option>
+                            <ng-option value="sold">sold</ng-option>
+                            <ng-option value="outOfService">OutOfService</ng-option>
```
Comment (FH-sandeepkaur1) [inline src/app/pages/reports/fleet/vehicle-report/summary/summary.component.html]: Please add spacing in OutOfService text. 
image

**Review by FH-sandeepkaur1 — CHANGES_REQUESTED** [review]
Please review changes

**Review by FH-sandeepkaur1 — APPROVED** [review]

File: src/app/pages/reports/fleet/fuel-report/summary/summary.component.html [hunk 2]
```diff
@@ -1,122 +1,226 @@
...
+    <div class="inner-wrapper">
+        <section role="main" class="content-body pt-0 pr-0 pl-1">
+            <header class="page-header pr19 pl11">
+                <div class="row" style="padding-top:10px;">
+
+                    <!--  
+                                   <div class="col-lg-3 col-md-3">
+                                <div class="input-daterange input-group">
+                                  <span class="input-group-prepend">
+                                    <span class="input-group-text">
```
Comment (FH-sandeepkaur1) [inline src/app/pages/reports/fleet/fuel-report/summary/summary.component.html]: Please remove commented code

**Review by FH-sandeepkaur1 — CHANGES_REQUESTED** [review]

File: src/app/pages/reports/fleet/fuel-report/summary/summary.component.html (hunk 2 above)
Comment (saurkarankita) [reply src/app/pages/reports/fleet/fuel-report/summary/summary.component.html]: Okay Sandeep

**Pull request merged**